examples/c++/multiply/example.cu

```diff
@@ -142,17 +130,10 @@ int main(int argc, char** argv)
     std::cerr << "Failed to get GPU temperature." << std::endl;
   }
   auto start_time = std::chrono::high_resolution_clock::now();
-
   // Benchmark loop
   for (int i = 0; i < repetitions; i++) {
     vector_mult(device_in1, device_in2, device_out, vector_size);
```

Comment:
We should change this to use the `vec_ops` operations from icicle and not the functions in the file at the top

Reply:
done